pkg/scheduler/algorithm/priorities/util/non_zero.go

```diff
@@ -32,20 +32,34 @@ const DefaultMilliCPURequest int64 = 100 // 0.1 core
 // DefaultMemoryRequest defines default memory request size.
 const DefaultMemoryRequest int64 = 200 * 1024 * 1024 // 200 MB
 
+// DefaultMilliCPURequestFraction defines default milli cpu request fraction.
+const DefaultMilliCPURequestFraction = 0.1
+
+// DefaultMemoryRequestFraction defines default memory request fraction.
+const DefaultMemoryRequestFraction = 0.1
+
 // GetNonzeroRequests returns the default resource request if none is found or what is provided on the request
-// TODO: Consider setting default as a fixed fraction of machine capacity (take "capacity v1.ResourceList"
-// as an additional argument here) rather than using constants
-func GetNonzeroRequests(requests *v1.ResourceList) (int64, int64) {
+func GetNonzeroRequests(requests *v1.ResourceList, capacity *v1.ResourceList) (int64, int64) {
```

Comment:
honestly, I'm a little confuse about this `TODO`. If we use 0.1 as fraction, there'll be 3.2 cores as default for a 32 cores host, that's a little big/many to me :)

Reply:
Thanks for your review! I'm not quite sure about the value, so needs some suggestions.
The alternative value is 0.1/32 if hosts have an average 32 cores, that will be consistent with the origin 0.1 core. WDYT @k82cn

Reply:
hm... would you help to check who adds this comments and ping him for more suggestion? Honestly, a constant is OK to me for this case.

Reply:
ping @hongchaodeng, you're the owner of the `TODO`. Please give suggestion on the default fraction value, thanks!